collections/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.19
 
 require (
 	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20221207205747-f3be41836f4d
+	github.com/hashicorp/go-multierror v1.1.1
```

Comment:
could we add a todo to replace this with golang std errors when we migrate to go 1.20.

Reply:
added